src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.CompletionListUpdater.cs

```diff
@@ -48,6 +51,9 @@ private sealed class CompletionListUpdater
 
             private readonly Action<IReadOnlyList<(RoslynCompletionItem, PatternMatch?)>, string, IList<RoslynCompletionItem>> _filterMethod;
 
+            private bool ShouldSelectSuggestionItem
+                => _snapshotData.DisplaySuggestionItem && _filterText.Length > 0;
```

Comment:
not sure i understand the concept here.  We want to select the suggestion item whnever we're displaying it, and there is some filter text?  that seems odd to me.

Reply:
perhaps ShouldSelectSuggestionItemWhenAllOtherItemsFilteredOut ?

Reply:
> ShouldSelectSuggestionItemWhenAllOtherItemsFilteredOut

Yup, that makes more sense